app/src/main/java/com/droidcba/kedditbysteps/features/news/adapter/NewsDelegateAdapter.kt

```diff
@@ -22,16 +27,17 @@ class NewsDelegateAdapter : ViewTypeDelegateAdapter {
         holder.bind(item as RedditNewsItem)
     }
 
-    class NewsViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
+    inner class NewsViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
             parent.inflate(R.layout.news_item)) {
 
         fun bind(item: RedditNewsItem) = with(itemView) {
-            //Picasso.with(itemView.context).load(item.thumbnail).into(img_thumbnail)
```

Comment:
Please don't remove this comment as intentionally is there for tutorial purpose.